AltStore/SceneDelegate.swift

```diff
@@ -139,7 +141,26 @@ private extension SceneDelegate
                 DispatchQueue.main.async {
                     NotificationCenter.default.post(name: AppDelegate.addSourceDeepLinkNotification, object: nil, userInfo: [AppDelegate.addSourceDeepLinkURLKey: sourceURL])
                 }
+            
+            case "jitenable":
+                let queryItems = components.queryItems?.reduce(into: [String: String]()) { $0[$1.name.lowercased()] = $1.value } ?? [:]
+                guard let jitdebugURLString = queryItems["url"] else { return }
                 
+                DispatchQueue.main.async {
+                    let v = minimuxer_to_operation(code: 1)
+                    
+                    do {
+                        var x = try debug_app(app_id: jitdebugURLString)
+                        switch x {
+                        case .Good: print(jitdebugURLString)
+                        case .Bad(let code): minimuxer_to_operation(code: code)
+                        }
+                    } catch Uhoh.Bad(let code) {
+                        minimuxer_to_operation(code: code)
```

Comment:
Might be good to alert the user if this happens

Reply:
Ngl that came with the code as I got this from a jitenable swift file. Maybe that should be changed also then. Super new at swift so not sure how to handle this.

Reply:
I think you could use a UIAlertController